electrode-ota-server-model-acquisition/test/acquisition-test.js

```diff
@@ -336,10 +338,9 @@ describe('model/acquisition', function () {
                     packageHash: "ABCD",
                     clientUniqueId
                 }).then(result => {
-                    expect(result.updateAppVersion).true;
```

Comment:
this one line is the whole point of this test.  test says `test updateAppVersion`.  put this back.

Reply:
reverted this change.
modified `getNewestApplicablePackage()` method to do one of the following:
> match by tags && version
> match by version on untagged packages
> match by tag
> by default return latest release on untagged packages

now, the above test passes.